extension/src/commands.ts

```diff
@@ -16,7 +16,7 @@ export function registerCommands(context: vscode.ExtensionContext){
 		}
 		catch (err) {
 			if (err){
-				let msg = localize("autolispext.help.commands.openWebHelp", "Failed to load the webHelpAbstraction.json file");
+				let msg = AutoLispExt.localize("autolispext.help.commands.openWebHelp", "Failed to load the webHelpAbstraction.json file");
```

Comment:
It (and othere similar changes in this PR) will break the Localization, see commit: https://github.com/Autodesk-AutoCAD/AutoLispExt/commit/fc630f78d3afefbf7842f1e596e1260e8f48af12

Reply:
Interesting I never saw this, I'll go read the documentation on vscode-nls again, I must have read it wrong or don't understand the scope of what I did read about not recreating this object.

Reply:
I think the actual problem has to do with our gulp file using `vscode-nls-dev` and that code using this regex
   `/^\s*(["'])vscode-nls\1\s*$/;` 
Which kicks out any file that doesn't directly import the `vscode-nls` reference.

I maintain that we have calls to `nls.config()` all over our code and according to everything I've read, this should not be happening. Per the documentation,  the `extension.ts` should own defining the `config()`

All other files should be importing `nls-config` because of the `vscode-nls-dev` anomaly and be creating their own localize reference using `loadMessageBundle()`

The `localize()` within the *context.ts* is entirely useless because of the limitation imposed by vscode-nls-dev, should be removed and purged from all existing documents; which I will add to this PR.

Reply:
FYI, purged `AutoLispExt.localize()` from all the drawings and used the `nls.loadMessageBundle()` to populate our `const` in those files. I tested the project commands responses previously using the AutoLispExt accessor and with Spanish; they popped up in Spanish as expected using a fresh VSIX compile. Can we assume they are functioning properly with `loadMessageBundle` based on that?

Reply:
Yes I built your code and tested against some of them. It looks properly translated.